Directory.Solution.targets

```diff
@@ -0,0 +1,251 @@
+<!--
+  Specifies build logic when building a solution file.
+-->
+<Project>
+  <PropertyGroup>
+    <!--
+      Set some default property values unless otherwise specified.
+    -->
+    <VSTestNoLogo Condition="'$(VSTestNoLogo)' == ''">true</VSTestNoLogo>
+    <VSTestVerbosity Condition="'$(SYSTEM_DEBUG)' == 'true'">detailed</VSTestVerbosity>
+    <VSTestVerbosity Condition="'$(VSTestVerbosity)' == ''">quiet</VSTestVerbosity>
+    <VSTestSetting Condition="'$(VSTestSetting)' == '' And '$(CI)' == 'true'">$([System.IO.Path]::Combine('$(MSBuildThisFileDirectory)', 'build', 'xunit.runsettings'))</VSTestSetting>
+    <VSTestResultsDirectory Condition="'$(VSTestResultsDirectory)' == ''">$([System.IO.Path]::Combine('$(MSBuildThisFileDirectory)', 'TestResults'))</VSTestResultsDirectory>
+    <VSTestLogger Condition="'$(VSTestLogger)' == ''">console%3Bverbosity=$(VSTestVerbosity)</VSTestLogger>
+
+    <!--
+      Build tests for all supported target frameworks by default, otherwise build tests for just the specified target framework.
```

Comment:
This is not really feedback, but just a shout out how much these comments are appreciated. 

MSBuild doesn't always have a clear correlation as to why a certain value is imported, so this is incredibly helpful.